upgrade-advisor.php

```diff
@@ -27,6 +27,11 @@ function isThere($setting) {
 
     $root_server_settings = json_decode(get(getHelplineBMLTRootServer() . "/client_interface/json/?switcher=GetServerInfo"));
 
+    if (strpos(getHelplineBMLTRootServer(), 'index.php')) {
+        echo "\n\nYour root server points to index.php. Please make sure to set it to just the root directory.";
```

Comment:
Shouldn't be a need to line break.  It will never show more than one error at a time.

Reply:
darn i missed those, had them there for testing to have some spacing, between tests